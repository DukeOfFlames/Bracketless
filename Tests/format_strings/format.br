START
°list = {1, 2, 3, 4}
°x = 777
#drucke{"hi"}
#drucke{"'x'"}
#drucke{"...'list'..."}
#drucke{"'x' 'list' 'x'"}
END