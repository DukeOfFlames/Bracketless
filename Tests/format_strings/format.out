"hi"
"777"
"...{1, 2, 3, 4}..."
"777 {1, 2, 3, 4} 777"
